.changeset/fifty-pears-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/test': minor
+---
+
+Options for `testModel.getCoverage()` and `testModel.testCoverage()` can now be provided to filter which state nodes should be covered by the tests.
```

Comment:
We could mention here how actually one is supposed to filter state nodes - either by mentioning `filter` option or linking to the docs.

Reply:
Will do that pre-release (and once the docs are published)